index.html

```diff
@@ -54,7 +56,7 @@
       j.async = true;
       j.src = '//www.googletagmanager.com/gtm.js?id=' + i + dl;
       f.parentNode.insertBefore(j, f);
-    })(window, document, 'script', 'dataLayer', 'GTM-K3K9K9');
+    })(window, document, 'script', 'dataLayer', 'GTM-K3K9K9', navigator.doNotTrack);
```

Comment:
@kpeckett It does look better to pass another argument and check for it inside of this anonymous function. I could make it like you suggested but it'd have the same effect.